### PR TITLE
Capitalise sentences.

### PR DESCRIPTION
**Pull request by cat-git**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ The provision script adds an virtual ip on eth0:0 bound to 192.168.1.6, and sets
 
 `vagrant up && vagrant ssh` to shell in
 
-you can throw some traffic at the interface by telnetting to it. Something like this will do it.
+You can throw some traffic at the interface by telnetting to it. Something like this will do it.
 
-open a new shell and run `nc -l 192.168.1.6 -p 12345` to listen on that interface
+Open a new shell and run `nc -l 192.168.1.6 -p 12345` to listen on that interface
```
Comment (cmstrickland) [inline Readme.md]: You could have added a full-stop

File: Readme.md [hunk 2]
```diff
@@ -6,12 +6,12 @@ The provision script adds an virtual ip on eth0:0 bound to 192.168.1.6, and sets
...
 
-from another shell `telnet 192.168.1.6 12345`
+From another shell `telnet 192.168.1.6 12345`
```
Comment (cmstrickland) [inline Readme.md:14]: also here

File: Readme.md [hunk 3]
```diff
@@ -6,12 +6,12 @@ The provision script adds an virtual ip on eth0:0 bound to 192.168.1.6, and sets
...
 
-type some jank into the second shell, you should see it coming in on the first one 
+Type some jank into the second shell, you should see it coming in on the first one
```
Comment (cmstrickland) [inline Readme.md]: here

File: Readme.md [hunk 4]
```diff
@@ -6,12 +6,12 @@ The provision script adds an virtual ip on eth0:0 bound to 192.168.1.6, and sets
...
+Open a new shell and run `nc -l 192.168.1.6 -p 12345` to listen on that interface
 
-from another shell `telnet 192.168.1.6 12345`
+From another shell `telnet 192.168.1.6 12345`
 
-type some jank into the second shell, you should see it coming in on the first one 
+Type some jank into the second shell, you should see it coming in on the first one 
 
-from a third shell `iptables -L -v` should show a packet count on the user chain
+From a third shell `iptables -L -v` should show a packet count on the user chain
```
Comment (cmstrickland) [inline Readme.md]: and here

**Comment by cat-git** [comment]
I have added some full stops.

**Pull request merged**